src/scripts/lychee.js

```diff
@@ -144,6 +144,12 @@ lychee.logout = function() {
 
 }
 
+lychee.restart = function() {
+
+	document.location.href = ''
```

Comment:
There's a redirect to the root when the current album is private. Also happens after a reload when the session expired: https://github.com/electerious/Lychee/blob/master/src/scripts/album.js#L58 Not sure if there're any edge cases that might cause problems.